apps/server/src/modules/deletion/services/deletion-log.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@nestjs/common';
+import { EntityId } from '@shared/domain';
+import { ObjectId } from '@mikro-orm/mongodb';
+import { DeletionLogRepo } from '../repo';
+import { DeletionLog } from '../domain/deletion-log.do';
+import { DeletionDomainModel } from '../domain/types/deletion-domain-model.enum';
+import { DeletionOperationModel } from '../domain/types/deletion-operation-model.enum';
+
+@Injectable()
+export class DeletionLogService {
+	constructor(private readonly deletionLogRepo: DeletionLogRepo) {}
+
+	async createDeletionLog(
+		deletionRequestId: EntityId,
+		domain: DeletionDomainModel,
+		operation: DeletionOperationModel,
+		modifiedCounter: number,
+		deletedCounter: number
```

Comment:
Please make it `modifiedCount` and `deletedCount`, respectively.

Reply:
done